infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationStreamNameTableManager.java

```diff
@@ -95,6 +102,55 @@ public LogReplicationStreamNameTableManager(CorfuRuntime runtime, String pluginC
         return readStreamsToReplicateFromTable();
     }
 
+    public void addStreamsToInfoTable(Set<UUID> streamIdSet) {
+        try {
+            Table<TableInfo, Namespace, CommonTypes.Uuid> streamsInfoTable = corfuStore.openTable(CORFU_SYSTEM_NAMESPACE,
```

Comment:
wouldnt it be easier to keep the table schema as uuid, TableInfo?.  This schema seems a bit non-intuitive.